web/forms.py

```
@@ -1787,3 +1792,42 @@ class StudyGroupForm(forms.ModelForm):
+            "description",
+            "features",
+            "price_monthly",
+            "price_yearly",
+            "billing_period",
+            "stripe_monthly_price_id",
+            "stripe_yearly_price_id",
+            "is_active",
+            "is_popular",
+            "order",
+        ]
+        widgets: ClassVar[Dict[str, Any]] = {
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)